.github/actions/openvino_provider/action.yml

```diff
@@ -4,28 +4,35 @@ inputs:
   revision:
     description: 'Version of OpenVINO to provide - latest_available | HEAD | specific commit hash'
     required: true
+  artifacts_source:
+    description: "From where to take artifacts: postcommit | s3"
+    default: "postcommit"
+    required: false
   branch_name:
     description: 'Branch of OpenVINO to take the revision from if no specific hash was provided. 
                   Taken from github context by default'
     required: false
   ov_artifact_name:
     description: "Name under which to upload provided OpenVINO build artifacts, set automatically by default"
     required: false
+  platform:
+    description: "Platform for which to get artifacts: centos7_x86_64 | debian10_armhf | rhel8_x86_64 | ubuntu20_arm64 | 
+      ubuntu20_x86_64 | ubuntu22_x86_64 | ubuntu24_x86_64 | macos_12_6_arm64 | macos_12_6_x86_64 | windows_x86_64"
+    required: false
```

Comment:
why is it not required? what platform will be used if this field is not?

Reply:
It is not required when package_url is set, if both are not set - the action will fail
package_url will be removed once we have mapping.json, it's present only to support RCs and releases until this is done